src/app/locale/translations/index.ts

```diff
@@ -1,3 +1,3 @@
-import DEFAULT_TRANSLATIONS from './en.json';
+import FALLBACK_TRANSLATIONS from './en.json';
 
-export { DEFAULT_TRANSLATIONS };
+export { FALLBACK_TRANSLATIONS };
```

Comment:
```suggestion
export { default as FALLBACK_TRANSLATIONS } from './en.json';
```

Reply:
I realised I couldn't do this because the JSON is exported as `export =` not as `default`. So I have to import and export in two separate steps.